clayui.com/content/docs/contributing.md

```diff
@@ -1,5 +1,4 @@
 ---
 title: 'Contributing'
 order: 4
-redirect: 'https://github.com/liferay/clay/blob/master/CONTRIBUTING.md'
```

Comment:
Only one other file has a `redirect` property in the frontmatter (`content/docs/index.md`), so perhaps this feature is just broken.

One thing to note about it though is it has `layout: 'redirect'`; maybe that is what's needed to make this work.